tests/cxx/java_like/Account.cpp

```diff
@@ -22,11 +22,9 @@ void Account::transferTo(Account *target, int amount)
 //@ requires AccountPred(this, ?b) &*& AccountPred(target, ?t);
 //@ ensures AccountPred(this, b-amount) &*& AccountPred(target, t+amount);
 {
-    //@ open AccountPred(this, _);
-    //@ close AccountPred(this, _);
+    //@ AccountNotNull(this);
```

Comment:
Is this explicit call necessary? Does the autolemma not trigger?

Reply:
The autolemma does indeed not trigger

Reply:
Perhaps try `[?f]AccountPred(?a, ?b)` instead of `AccountPred(?a, ?b)`?